packages/commonwealth/firebase-messaging-sw.js

```diff
@@ -0,0 +1,77 @@
+// NOTE: this needs to be done in .js due to a Typescript issue which is a bit of
+// a nightmare.  What's happening is that we can't use Typescript types as our
+// 'libs' configuration requires 'ServiceWorker' but that means that both 'dom'
+// and ServiceWorker types are available in two places which are wrong.  The dom
+// APIs are available to the service worker (which is wrong) and the ServiceWorker
+// APIs are available to React (the browser) which is also wrong.  Additionally,
+// the types for the service worker are actually wrong and 'showNotification'
+// does not appear to be in the ServiceWorker types.  To handle this properly
+// we would need to upgrade our Typescript to 5.4.x and create a dedicated
+// typescript package JUST for our service worker which I feel is a waste of time
+// since our service worker is just one file and less than 100 lines of code
+
+// Listen for 'push' events from the push service
+self.addEventListener('push', function (event) {
+  if (!event.data) {
+    console.warn('Push notification lacks data. Ignoring');
+    return;
+  }
+
+  const img = '/static/img/branding/common.png';
+
+  const data = event.data.json();
+  const options = {
+    body: data.body || data.comment_body || 'No body',
+    image: img,
+    icon: img, // Replace with the path to your notification icon
+    badge: img, // Replace with the path to your notification badge
```

Comment:
let's remove these 2 comments

Reply:
I removed the first one. I think the second one should stay.